docs/v2-system-addon/data_events.md

```diff
@@ -290,6 +290,15 @@ perf: {
   // topsites has yet to receive screenshots updates from the add-on code,
   // and is therefore just showing placeholder screenshots.
   "topsites_first_painted_ts": 5,
+
+  // How much longer the data took, in milliseconds, to be ready for display
+  // than it would have been in the ideal case. The user currently sees placeholder
+  // cards instead of real cards for approximately this length of time. This is
+  // sent when the first call of the component's `render()` method happens with
+  // `this.props.initialized` set to `false`, and the value is the amount of
+  // time in ms until `render()` is called with `this.props.initialized` set to `true`.
+  "highlights_data_late_by_ms": 67,
```

Comment:
As those two fields have been promoted to be keys in the "session.perf", let's also add them to the "data_dictionary.md" with some detailed descriptions.

Reply:
Also, since this has updated the session schema, let's make sure the update gets synced in `system-addon/test/schemas/pings.js`. Adding some unit tests accordingly would be perfect :)